packages/common/src/Paginator.vue

```diff
@@ -2,7 +2,7 @@
   <nav class="paginator" :key="urnString">
     <router-link
       v-if="urn"
-      :to="{ name: 'reader', params: { urn: urnString } }"
+      :to="{ name: 'reader', params: { urn: urnString }, query }"
```

Comment:
@paltman I pushed a small fix; can you please take a another pass through and fix any other instances where we would _always_ want to pass `query` to the `reader` route?